lib/sqlalchemy/testing/requirements.py

```diff
@@ -1579,6 +1580,11 @@ def check_lib():
 
         return exclusions.only_if(check_lib, "patch library needed")
 
+    @property
+    def gil_enabled(self):
+        """Test should run if the Python distribution is GIL-enabled."""
+        return bool(sysconfig.get_config_var("Py_GIL_DISABLED"))
```

Comment:
I believe the canonical way to check for this is with [`sys._is_gil_enabled()`](https://docs.python.org/3/library/sys.html#sys._is_gil_enabled).